README.md

```diff
@@ -1 +1,123 @@
-# Currency-explorer
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Awesome books ](#-weather-)
+  - [🛠 Built With ](#-built-with-)
+    - [Key Features ](#key-features-)
+    - [Tech Stack ](#tech-stack-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖 Weather <a name="about-project"></a>
+
+**Weather App** in this project, i create a weather app using React.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **[Search your city]**
+- **[Go to detail page]**
+- **[React Router Dom]**
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML | CSS | JS | ES6 | React
+
+- More details about the project can be seen in this [video](https://www.loom.com/share/18778594bc524f93bfeabd8d35a33ca3) 
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [live link](https://hasnatali1947.github.io/Weather-App/)
```

Comment:
- [ ] Great job working on the react capstone project. As per proeject requirements, you are supposed to deploy your app and add the live link to your README. 
**Live version**
```
- The student deploys the application using Render, Netlify or GitHub Pages.
- The project is deployed properly, the home and detail page works as expected.
- A link to the deployed project is present in the README.
```
Unfortunately, when I try to visit your webpage from the shared [link](https://hasnatali1947.github.io/Weather-App/), I get the the page as per on the screenshot below. Kindly fix this deployment issue.
![image](https://github.com/hasnatali1947/Weather-App/assets/62849872/ce8df941-719a-4e1f-87a0-633d6731e09c)